test/k8s-integration/main.go

```diff
@@ -598,6 +598,11 @@ func generateGKETestSkip(testParams *testParameters) string {
 		skipString = skipString + "|pvc.data.source"
 	}
 
+	// Snapshot and restore test fixes were introduced after 1.26 in PR#972.
+	if curVer.lessThan(mustParseVersion("1.26.0")) {
```

Comment:
Yeah just remove the version check altogether

Reply:
ok will fix now.

Reply:
Ok I am going to just try to do this directly in the cherry picks for the CVE's since this one isn't really necessary